### ISSUE TITLE
consider providing android package

### ISSUE THREAD

**Issue by igalic** [thelounge/thelounge#3206]
while the lounge works great on any mobile browser, it makes for a far more comfortable experience to have a standalone application 

### Feature Description

a WebView packed thelounge  would be independent from the users main browser which may be heavily used for, well, browsing.

**Comment by Zarthus** [comment]
Aren't there apps for android already that will permit opening any arbitrary web-page into a more isolated app? Why not use them?

**Comment by Jay2k1** [comment]
The fact that the main browser is used for browsing shouldn't make too much of a difference when you use the "add to home screen" feature which then opens an extra instance without browser UI.

**Comment by McInkay** [comment]
Yeah, "Add to home screen" does all of this functionality, and we won't be replicating that.

**Comment by cinderblock** [comment]
I for one would love a non-webpage based dedicated app (even if it's a web view)

**Comment by Jay2k1** [comment]
What do you think would be the benefit?

**Comment by cinderblock** [comment]
I prefer the notification integrations that seem available to "native" applications.

Full disclosure, I came across this issue looking for a standalone client for Windows/Mac/iOS. In general. I personally don't use android much. Sorry to hijack the issue.

Rocket.chat, as an example, has "native" clients for most devices. This is particularly nice with things like iOS push notifications working from private servers (yes, dependent on Rocket.Chat maintaining an Apple subscription).

**Comment by MiniDigger** [comment]
thelounge is a PWA, a progressive web app. On devices that support it, it will provide native push notifications (iirc everybody but apple implemented this).
If you open your instance in chrome or firefox on your android device, you will see a "Install The Lounge" button in the settings dropdown for that browser tab. This will install the PWA on your device and it will then look exactly like a native app. 
It even shows up separately in Digital Wellbeing on a Pixel which is really cool. 

So, bottom line, there is no benefit of packaging a "native" application since a PWA is already as close to a native app as it needs to be.

**Comment by cinderblock** [comment]
I totally missed the "Install" button on my browser. Didn't realize PWAs enabled this. Very cool. I guess the one thing that I'd still like as an iOS user is the integration there. But sounds like that's either dependent on an Apple developer account or Apple to improve push support. So maybe we can add another issue to track that (or just forget about it for now).

**Comment by xPaw** [comment]
> So maybe we can add another issue to track that.

No. Push Apple to support the web push specification, as every other vendor does.

**Comment by MiniDigger** [comment]
Whenever apple feels like deprecating their own custom implementation and using actual standards, it will work right away on iOS, so a tracking issue for the lounge isn't needed.
